docs/ensuring-repro/workflows/run-module-gha.md

```diff
@@ -0,0 +1,42 @@
+# Automated module testing
+
+To maintain module functionality over time, we use [GitHub Actions](https://docs.github.com/en/actions) (GHAs) to periodically run each module ("module testing GHA") with the goal of testing that it runs to completion without errors.
+
+!!! info
+    For more information about how we run modules to generate official results and OpenScPCA releases, please see our documentation on the `OpenScPCA-nf` workflow. <!-- openscpca-nf STUB_LINK -->
+
+Module testing GHAs are automatically run under these two circumstances:
+
+- When a pull request is filed with changes to any module files
+    - This GHA will need to pass without errors for [pull requests](../../contributing-to-analyses/pr-review-and-merge/index.md) to be approved
+- All modules are periodically run on a schedule to ensure they continue to pass tests as OpenScPCA grows
+
+To make GHAs run efficiently, we strongly encourage and generally require, when applicable, that GHAs run the module code using the [simulated test data](../../getting-started/accessing-resources/getting-access-to-data.md#accessing-test-data).
+This means that it's important to write your module code with sufficient flexibility to allow for test data to be used.
+For example, when reading input data into scripts, you should use arguments for input data files, allowing for test data paths to be used in these module testing GHAs.
```

Comment:
The test paths are designed to match real data paths, so no changes should be needed in most cases. 

```suggestion
This means that it's important to write your module code with sufficient flexibility to allow for test data to be used.
You should read in files from the `data/current` directory, which will be automatically directed to test data during GHA test runs. 
```

If you want to add more detail here, it might be that things like checks for numbers of cells might need to be adjusted, but these kinds of things will be specific to individual modules and I do not think it is something we will be able to have any general instructions about. 


I think a more general thing to say (maybe in the next section?) is that it is helpful for testing to have a single entry point to run the whole module. Or at least a set of well-labelled scripts. You do mention this later, but it probably makes sense to talk about here too.

Reply:
Added a couple spots about data to clarify this